src/screens/create-safe/containers/safe-creation.container.tsx

```diff
@@ -0,0 +1,129 @@
+            type='email'
+            placeholder={t('createSafeForm.placeHolders.email')}
+            label={t('createSafeForm.addBeneficiary')}
+            onChange={(e: any) => setEmail(e.target.value)}
+            errorMsg={t('createSafeForm.FormValidation.inValidEmail')}
+            error={showError && !FormValidator.isEmailValid(email)}
+          />
+        </FlexContainer>
+        <Spacer />
+        <StyledDropDown
+          label={t('createSafeForm.selectRecoveryMethod')}
+          options={createSafeController.getRecoveryMethods()}
+          onChange={() => 'changed'}
+        />
+        <Spacer />
+        <FlexContainer>
+          <StyledDropDown
+            label={t('createSafeForm.selectTheWalletStoreType')}
+            options={createSafeController.getWalletStoreTypes()}
+            onChange={(e: any) => createSafeController.setWalletStoreType(e.target.value)}
+            wide
+          />
+          {createSafeController.showSecretStoreType() && (
+            <StyledDropDown
+              label={t('createSafeForm.selectTheSecretStoreType')}
+              options={createSafeController.getSecretStoreType()}
+              onChange={(e: any) => createSafeController.setSecretStoreType(e.target.value)}
+              wide
+            />
+          )}
+          {createSafeController.showInstructionsStoreType() && (
+            <StyledDropDown
+              label={t('createSafeForm.selectTheInstructionStoreType')}
+              options={createSafeController.getSecretStoreType()}
+              onChange={() => 'changed'}
+              wide
+            />
+          )}
+        </FlexContainer>
+        <Spacer />
+
+        {createSafeController.selectedSecretStoreType === SecretStoreType.PrivateKey && (
+          <WideInput
+            type='text'
+            placeholder={t('createSafeForm.placeHolders.privateKey')}
+            label={t('createSafeForm.enterYourPrivateKey')}
+          />
+        )}
+        {createSafeController.selectedSecretStoreType === SecretStoreType.KeyStore && (
+          <WideInput
+            type='text'
+            placeholder={t('createSafeForm.placeHolders.secretStore')}
+            label={t('createSafeForm.secretStore')}
+          />
+        )}
+        {createSafeController.selectedSecretStoreType === SecretStoreType.SeedPhrases && (
+          <StyledChipInput seedPhraseList={seedPhrases} label={t('createSafeForm.enterSeedPhrases')} />
+        )}
+
+        <Spacer />
+        <StyledTextArea
+          label={t('createSafeForm.description')}
+          placeholder={t('createSafeForm.placeHolders.description')}
+          wide
+        />
+        <Spacer />
+        <Spacer />
+      </FormContainer>
+      <ButtonContainer>
+        <StyledButton label={{ text: 'Cancel', color: 'textLighter' }} variant='ghost' onClick={() => ''} />
+        <StyledButton label={{ text: 'Create' }} variant='primary' color='primaryGradient' onClick={handleSubmit} />
```

Comment:
Is this text supposed to be hard-coded? Line 124, 125

Reply:
added to JSON file👍